packages/syft/src/syft/service/network/node_peer.py

```diff
@@ -87,3 +198,10 @@ def guest_client(self) -> SyftClient:
 
     def proxy_from(self, client: SyftClient) -> SyftClient:
         return client.proxy_to(self)
+
+    def pick_highest_priority_route(self) -> NodeRoute:
+        final_route = self.node_routes[-1]
+        for route in self.node_routes:
+            if route.priority > final_route.priority:
+                final_route = route
+        return final_route
```

Comment:
```suggestion
        return max(self.node_routes, key=lambda r: r.priority)
```
more Pythonic and also works if `self.node_routes` is empty.